docs/types/reference-types.rst

```diff
@@ -165,31 +165,40 @@ always use one of the value types ``bytes1`` to ``bytes32`` because they are muc
     that you are accessing the low-level bytes of the UTF-8 representation,
     and not the individual characters.
 
-.. index:: ! bytes-concat
+.. index:: ! concat-functions
 
-.. _bytes-concat:
+.. _string-concat:
 
-``bytes.concat`` function
-^^^^^^^^^^^^^^^^^^^^^^^^^
+``<bytes/string>.concat`` function
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the string.concat function you can concatenate a variable number of ``string`` variables using ``string.concat``.
+The function returns a single ``string memory`` array that contains the contents of the arguments without padding.
+If you want to use parameters of other types, you need to convert them to ``string`` first.
 
-You can concatenate a variable number of ``bytes`` or ``bytes1 ... bytes32`` using ``bytes.concat``.
+Analogously, the bytes.concat function can concatenate a variable number of ``bytes`` or ``bytes1 ... bytes32`` using ``bytes.concat``.
```

Comment:
```suggestion
Analogously, the ``bytes.concat`` function can concatenate a variable number of ``bytes`` or ``bytes1 ... bytes32`` using ``bytes.concat``.
```

Reply:
Actually I think for both bytes.concat and string.concat you can also use array slices